apps/common/src/main/scala/org/lfdecentralizedtrust/splice/automation/RoundBasedRewardTrigger.scala

```diff
@@ -96,8 +161,18 @@ abstract class RoundBasedRewardTrigger[T <: RoundBasedTask: Pretty]()(implicit
 
 object RoundBasedRewardTrigger {
 
+  final case class RoundBasedTriggerState(
+      roundNumber: Long,
+      runAt: Instant,
```

Comment:
discussion: earliest time at which I can run for that round. I might run more often for that round if `workStillToBeDone` is true.